backend/app/repositories/custom_bot.py

```diff
@@ -57,6 +59,9 @@ def store_bot(user_id: str, custom_bot: BotModel):
         "ApiPublishmentStackName": custom_bot.published_api_stack_name,
         "ApiPublishedDatetime": custom_bot.published_api_datetime,
         "ApiPublishCodeBuildId": custom_bot.published_api_codebuild_id,
+        **_parse_generation_config_to_db_item(
```

Comment:
Could you replace with like following?
```py
"GenerationConfigParams": custom_bot.generation_config_params.model_dump()
```

The main reason is we'd like to keep consistency to align `EmbeddingParams`.

Reply:
top_p and temperature are float so we will need to parse decimal so that we do not get ddb write error

Reply:
I see. Could you please take some trouble to implement following so that succeeding contributors can easily expand models?

Firstly, we'd like to declare customized type so that we are not able to care the type conversion between float and decimal like:

```py
from decimal import Decimal
from pydantic.functional_serializers import PlainSerializer

# Declare customized float type
Float = Annotated[
    # Note: Before decimalization, apply str() to keep the precision
    float, PlainSerializer(lambda v: Decimal(str(v)), return_type=Decimal)
]

class ExampleParamModel(BaseModel)
  top_k: Float # Use customized float
  
m = ExampleParamModel(top_k=0.72)
print(type(m.model_dump()["top_k"])) # Decimal

d = {
  "top_k": 0.81
}
m = ExampleParamModel(**d)
print(type(m.top_k)) # float
```

- In this application, the `Float` can be declared in `repositories/models/common.py`, which should be newly created
- Import the Float type for each modules

I believe that the above approach can omit the `_parse_generation_config_to_db_item` and `_parse_generation_config_from_db_item`, which makes the code maintainable.

Reply:
Actually I implemented some attributes (e.g. `create_time` and `last_used_time`) as built-in float, but I'd like to refactor them if the above approach works.